### PR TITLE
update properties file

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Remove properties now distributed across the various properties file

# Testing this PR

Check that the server starts
https://trello.com/c/Tk1L87GB/107-point-to-review

### CODE REVIEW

**Comment by mtbc** [comment]
Testing in conjunction with:
- https://github.com/ome/omero-common/pull/17
- https://github.com/ome/omero-model/pull/25
- https://github.com/ome/omero-server/pull/30
- https://github.com/ome/omero-blitz/pull/39

**Comment by mtbc** [comment]
Weird CLI import exception:
```
2019-04-15 11:58:29,792 4548       [      main] ERROR  formats.importer.cli.CommandLineImporter - Error during import process.
java.util.MissingResourceException: Can't find bundle for base name omero, locale en_US
```

(Similar happens if I set `en_GB` instead.)

**Comment by mtbc** [comment]
I also got an error in `Blitz-0.log`,
```
SERVER IMPROPERLY CONFIGURED - CANNOT FIND /home/mtbc/src/openmicroscopy/dist/./lib/scripts/omero/import_scripts/Populate_ROI.py
```
but I'm used to seeing that, I doubt it's at all related to these PRs.

**Comment by mtbc** [comment]
(The import exception happens during the upgrade check.)

**Comment by mtbc** [comment]
With `--skip upgrade` I can import images then view them in the web client, including plates and big images.

**Comment by mtbc** [comment]
:+1: Your upgrade check fix https://github.com/ome/omero-blitz/commit/1115b2b8918222ac587841eef17f9a4d7bac2cf9 fixed the import exception for me: I can now use OMERO locally with your suite of PRs and in smoketesting it works fine. I can also connect to it with Insight. So, no remaining problems observed from my side.

**Comment by jburel** [comment]
--exclude

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-DEV-merge-push#1309](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/1309/). See the [console output](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/1309/consoleText) for more details.
Possible conflicts:
  - PR #5927 manics 'glacier2: support ice websockets for client connections'
    - etc/omero.properties
  - PR #5998 manics 'Glacier2 IceSSL properties can be overridden or extended'
    - etc/omero.properties

--conflicts

File: etc/omero.properties [hunk 1]
```diff
@@ -952,13 +114,16 @@ omero.client.browser.thumb_default_size=96
...
 Ice.IPv6=1
 
-### END
-
 #############################################
 ## Server product name for release artifacts
 #############################################
 product.name=OMERO.server
 
+# add the 2 db properties until we fix bin/omero db script
```
Comment (mtbc) [inline etc/omero.properties:121]: Once builds are green let's see if #6011 fixes this.

**Comment by jburel** [comment]
Due to issue with Travis, I have removed the last commit  so we can merge few PRs and unlock other work

**Pull request merged**